### PR TITLE
fix some cs issues in the 3.7. branch

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request after Commit https://github.com/joomla/joomla-cms/commit/fc41a23a9e8740c7864eaee166e1a089940cec97
### Summary of Changes

Some CS after the last commit: https://github.com/joomla/joomla-cms/commit/fc41a23a9e8740c7864eaee166e1a089940cec97
### Testing Instructions
- Code review
- double check that #11246 is not broken.
- check the isis pagenavigation on desktop and mobile.
### Documentation Changes Required

None

### CODE REVIEW

**Comment by wilsonge** [comment]
One test should be sufficient for this :)

**Comment by brianteeman** [comment]
Looks like a lot more than codestyle to me. CS is changing tabs to spaces and new lines. This is different code

File: administrator/templates/isis/html/layouts/joomla/pagination/link.php [hunk 1]
```diff
@@ -9,21 +9,23 @@
...
 	$liClass = $options->get('liClass', '');
 	$addText = $options->get('addText', '');
 }
-else
-{
-	$liClass = $addText = '';
-}
 
-$display = $item->text;
+$item->text .= $addText ? $addText : '';
```
Comment (bertmert) [inline administrator/templates/isis/html/layouts/joomla/pagination/link.php]: Too early here. See line `$item->text = JText::_('JPREVIOUS');` later.

File: administrator/templates/isis/html/layouts/joomla/pagination/link.php [hunk 2]
```diff
@@ -9,21 +9,23 @@
 
 defined('JPATH_BASE') or die;
 
+use Joomla\Registry\Registry;
```
Comment (bertmert) [inline administrator/templates/isis/html/layouts/joomla/pagination/link.php]: I did not see the benefit of importing/aliasing the class here if it's only used once and may be not used at all later on.
I could not find a related Joomla CS rule. That's why I called it directly with `new` when we need it.

**Comment by bertmert** [comment]
As an aside: Lots of these CS issues and code are taken over 1:1 from /layouts/pagination/link.php and links.php. Should be fixed there, too(?)

**Pull request closed without merging**